src/types/functions.ts

```diff
@@ -36,13 +36,15 @@ export interface AddFunctionDescription {
   args: ArgDefinition[];
   returns: [ArgType];
   isExported?: boolean;
+  hidden?: boolean;
 }
 
 export interface FunctionDescription extends AddFunctionDescription {
   minArgRequired: number;
   maxArgPossible: number;
   nbrArgRepeating: number;
   getArgToFocus: (argPosition: number) => number;
+  hidden?: boolean;
```

Comment:
not useful since it extends `AddFunctionDescription` which has the same property definition